src/libs/reportUtils.js

```diff
@@ -71,10 +71,24 @@ function findLastAccessedReport(reports) {
     return _.last(sortReportsByLastVisited(reports));
 }
 
+/**
+ *
+ * @param {String} chatType
+ * @returns {Boolean}
+ */
+function isDefaultRoom(chatType) {
```

Comment:
might be cleaner to pass the report object itself (since in all your calls you're passing report.chatType anyway)

Reply:
The one place I couldn't figure this out for was in [getSimplifiedReportObject()](https://github.com/Expensify/Expensify.cash/blob/34ef6a8d63ec6bc2c7d5494cd3555647c166ab1f/src/libs/actions/Report.js#L181) which eventually leads to [here where I call on just chatType](https://github.com/Expensify/Expensify.cash/blob/34ef6a8d63ec6bc2c7d5494cd3555647c166ab1f/src/libs/actions/Report.js#L143)

I guess I could just call something like `isDefaultRoom({chatType})`, would that be cleaner?

Reply:
Yup, that sounds like a good workaround to me 👍  It's just that one spot where we'll have `isDefaultRoom({chatType})` , while we'll be able to shorten things in other places from `report.chatType` => `report`.